Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/PhysicsSetupManipulatorCommandCallback.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Editor/Plugins/Ragdoll/PhysicsSetupManipulatorCommandCallback.h>
+#include <Editor/Plugins/Ragdoll/PhysicsSetupManipulatorBus.h>
+
+namespace EMotionFX
+{
+    bool PhysicsSetupManipulatorCommandCallback::Execute(
+        [[maybe_unused]] MCore::Command* command, [[maybe_unused]] const MCore::CommandLine& commandLine)
+    {
+        PhysicsSetupManipulatorRequestBus::Broadcast(&PhysicsSetupManipulatorRequests::OnUnderlyingPropertiesChanged);
+        return true;
+    }
+
+    bool PhysicsSetupManipulatorCommandCallback::Undo(
+        [[maybe_unused]] MCore::Command* command, [[maybe_unused]] const MCore::CommandLine& commandLine)
+    {
+        PhysicsSetupManipulatorRequestBus::Broadcast(&PhysicsSetupManipulatorRequests::OnUnderlyingPropertiesChanged);
+        return true;
+    }
```

Comment:
Why don't we do the work here directly? Using using a callback that fires an ebus event is one event too much in my eyes. If we can't do the work here as we don't have access to the data, we should add the callbacks on the places where we have access to the data.

Reply:
Yeah I guess that's a bit much. Simplified in the latest version. 

![image](https://user-images.githubusercontent.com/82226198/168253427-2d9d1eed-9557-46b4-86f0-06a6f33095b7.png)